contracts/tasks/governance.js

```diff
@@ -94,8 +94,46 @@ async function proposal(taskArguments, hre) {
   console.log("  actions:  ", JSON.stringify(actions, null, 2));
 }
 
+// Dumps the governor address for all the known contracts in the system.
+async function governors() {
+  const cOUSDProxy = await ethers.getContract("OUSDProxy");
+  const cVaultProxy = await ethers.getContract("VaultProxy");
+  const cChainlinkOracle = await ethers.getContract("ChainlinkOracle");
```

Comment:
OracleRouter

Reply:
Removed ChainlinkOracle. And OracleRouter does not have a governor so I did not include it here.